src/main/webapp/app/shared/orion/orion-button/orion-button.component.html

```diff
@@ -8,6 +8,8 @@
     (click)="handleClick()"
 >
     <fa-icon [hidden]="!buttonLoading" [spin]="true" [icon]="faCircleNotch"></fa-icon>
-    <span *ngIf="!buttonLoading" class="intellij-logo"></span>
+    <fa-icon [hidden]="buttonType !== OrionButtonType.Reload" [icon]="faRotateRight"></fa-icon>
+    <fa-icon [hidden]="buttonType !== OrionButtonType.Submit" [icon]="faUpload"></fa-icon>
+    <fa-icon [hidden]="buttonType !== OrionButtonType.Download" [icon]="faDownload"></fa-icon>
```

Comment:
why do you use `!==` here?

Reply:
the icons only appear when the buttontype is matched as the condition specifies when the icon is hidden